thirdeye/thirdeye-frontend/tests/acceptance/edit-alert-test.js

```diff
@@ -3,15 +3,44 @@ import moduleForAcceptance from 'thirdeye-frontend/tests/helpers/module-for-acce
 
 moduleForAcceptance('Acceptance | edit alert');
 
-const EDIT_BUTTON = '.te-search-results__edit-button';
+const EDIT_BUTTON = '.te-search-results__edit-button a';
+const METRIC_NAME = `#select-metric`;
+const GRANULARITY = `#select-granularity`;
+const ALERT_NAME = `#anomaly-form-function-name`;
+const STATUS = '.te-toggle--form span';
 
-test('visiting /edit-alert', function(assert) {
+test('visiting /manage/alerts/edit and checking that fields render correctly', function(assert) {
   visit('/manage/alerts');
 
   andThen(function() {
     assert.equal(
       currentURL(),
       '/manage/alerts',
       'manage alerts url is correct');
+    click(find(EDIT_BUTTON).get(1));
```

Comment:
fyi, `andThen` , `click` are both async helpers, you don't have to nest them 
https://guides.emberjs.com/v2.14.0/testing/acceptance/#toc_asynchronous-helpers

Reply:
I placed the `click` statement outside of `andThen` and the test failed because it couldn't find the `EDIT_BUTTON`.

Reply:
ah, probably due to the `find` being synchronous. 👍

Reply:
Alrighty. I'll submit this as is. Thanks for comment.